### PR TITLE
Bug update

### LINKED ISSUE #113: Change bug movement to be more tank like

### PR DESCRIPTION
**Pull request by Galaxy25**
Closes #113

### CODE REVIEW

File: Assets/Scripts/Enemies/EnemyAI.cs [hunk 1]
```diff
@@ -13,12 +13,14 @@ public abstract class EnemyAI : MonoBehaviour
     protected GameObject player;
     protected AIPhysics aiPath;
     protected Rigidbody2D rb;
+    protected Animator anim;
```
Comment (BrandonPacewic) [inline Assets/Scripts/Enemies/EnemyAI.cs:16]: What does anim stand for? Animation or Animator?

Comment (Galaxy25) [reply Assets/Scripts/Enemies/EnemyAI.cs:16]: animator

**Review by BrandonPacewic — APPROVED** [review]
Looks good 👍

**Comment by BrandonPacewic** [comment]
I'm getting ready to merge and updating the change log - please @me if any other changes are pushed.

**Pull request merged**